src/main/java/org/sagebionetworks/web/client/jsinterop/SRC.java

```diff
@@ -25,6 +25,7 @@
 		public static ReactFunctionComponent<IconSvgProps> IconSvg;
 		public static ReactFunctionComponent<EntityTypeIconProps> EntityTypeIcon;
 		public static ReactFunctionComponent<UserProfileLinksProps> UserProfileLinks;
+		public static ReactFunctionComponent<SkeletonButtonProps> SkeletonButton;
```

Comment:
Should we expose this in SRC?

Reply:
You mean it's available in the UMD build but not to portals?  If that's the case, I wouldn't worry about it (we can export once we have a case that needs it)